client/src/inside/components/page/test/ehr-helper.spec.js

```diff
@@ -0,0 +1,227 @@
+import should from 'should'
+import EhrHelpV2, { LEAVE_PROMPT } from '../ehr-helper'
+import store from '../../../../store/index'
+import { getPageKeys } from '../../../../helpers/test/testHelper'
+
+// the following pageKey must contain one more table elements in its contents, 
+// so that it can be used in 
+const tablePageKey = getPageKeys()[3] 
+// the following pageKey must contain one more form elements in its contents, 
+// so that it can be used in 
+const formPageKey = getPageKeys()[0]
+
+const _setUpWindowObj = () => {
+  delete window.confirm
+  window.confirm = jest.fn()
+}
+
+let ehrHelper, formKey, tableKey = 'table', router = {}
+router.go = jest.fn()
+
+describe('ehr-helper tests', () => {
+  it('properly instantiates class', () => {
+    ehrHelper = new EhrHelpV2(this, store, formPageKey)
+    should.exist(ehrHelper)
+  })
+
+  it('isV2', () => {
+    ehrHelper.isV2().should.equal(true)
+  })
+
+  it('getPageKey', () => {
+    const result = ehrHelper.getPageKey()
```

Comment:
Make this test independent of other tests. So instantiate the ehr helper inside this test.  Perhaps create a new describe section and use a before test method.